### ISSUE TITLE
Add a feature for copying selected files without FastCopy

### ISSUE THREAD

**Issue by ccppuser** [ccppuser/file-list-exporter#1]
**Is your feature request related to a problem? Please describe.**
Using FileListExporter, I exported a txt file for FastCopy and try to copy files using FastCopy.
After the copying is finished, the result is not I expected.
I had selected few folders in a parent folder.
But after copying, those selected folders were not in the parent folder.
They were in the root folder, which I had chose for destination.
In conclusion, I cannot maintain original folder hierarchy using FastCopy.

**Describe the solution you'd like**
Please provide one more button to copy selected files, without using FastCopy.
But it must maintain original forder hierarchy.

**Describe alternatives you've considered**
None.

**Additional context**
None.
